.github/workflows/deploy.yaml

```diff
@@ -35,4 +35,6 @@ jobs:
           file: Dockerfile
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.meta.outputs.labels }}
+      - name: autodeploy
```

Comment:
Would be good to move this to a separate PR. It's a self-contained functionality, and once we merge this two-line PR, then everything else should be easier.